datanode/src/storage_api.py

```diff
@@ -89,11 +90,47 @@
 @webapp.route('/', methods=['GET'])
 @webapp.route('/status', methods=['GET'])
 def Status():
-  # just a quick status checker, not really a health check
```

Comment:
We discussed adding another endpoint that does a lot less (like the current status) so we can treat it like a heartbeat and hit it every few seconds. The /status we may hit every 5-10 minutes or so.